UpdateProfile.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title></title>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+  </head>
+  <body>
+    <div class="container">
+      <div class="post-form">
+        <div class="form-row mb-lg-4">
+          	<img src="Jaime.jpg" class="rounded-circle" style="height:200px; margin-left:15px; padding: 10px;" />
+
+            <div class="form-column">
+              <div class="form-group form-inline">
+                <label for="exampleInputEmail1" class="col-lg-3 col-form-label" style="padding:10px">Name:</label>
+                <input type="text" class="form-control" id="Name" placeholder="Name">
+              </div>
+
+              <div class="form-group form-inline">
+                <label for="exampleInputEmail1" class="col-lg-3 col-form-label" style="padding:10px" >Contact Number:</label>
+                <input type="text" class="form-control" id="exampleInputEmail1" placeholder="0917XXXXX">
+              </div>
+            </div>
+            </div>
+
+            <div class="form-row mb-lg-5">
+        			<label for="facebook-name" class="col-lg-3 col-form-label">Additonal Details:</label>
```

Comment:
"Additional" is misspelled HAHA

Reply:
This should also be a different font (Alegreya Sans, you can check out more details sa Figma by clicking on the word and checking the Code tab). You change this through an external CSS file (like the one I did in my other branch)